versioned/persist/tx/src/main/java/org/projectnessie/versioned/persist/tx/TxDatabaseAdapter.java

```diff
@@ -591,6 +688,21 @@ protected int entitySize(KeyWithType entry) {
     return toProto(entry).getSerializedSize();
   }
 
+  @Override
+  public Stream<RefLog> refLog(Hash offset) throws RefLogNotFoundException {
```

Comment:
This method should be in the "block" with the api implementations above

Reply:
done